src/pages/admin/team/components/team-page-toolbar/TeamPageToolbar.test.tsx

```diff
@@ -288,6 +354,35 @@ describe('TeamPageToolbar', () => {
         });
     });
 
+    describe('TeamPageToolbar branch coverage for lines 189-191, 215', () => {
```

Comment:
You do not need to specify for which lines those tests are
Either remove describe or change it's title to exlude `branch coverage for lines 189-191, 215`